src/pybind/mgr/rook/module.py

```diff
@@ -55,7 +56,8 @@ def wrapper(*args, **kwargs):
 def write_completion(on_complete,  # type: Callable
                      message,  # type: str
                      mgr,
-                     calc_percent=None  # type: Optional[Callable[[], RookCompletion]]
+                     # type: Optional[Callable[[], RookCompletion]]
+                     calc_percent=None
```

Comment:
Hm. That probably prevents the type annotation from being checked. 

```suggestion
                     calc_percent: Optional[Callable[[], RookCompletion]]=None
```

Reply:
withe the suggested change i hava "invalid syntax ..." .. i leave as it was